lib/resources/qubic_js.dart

```diff
@@ -128,28 +128,33 @@ class QubicJs {
 
   Future<SignedTransaction> createTransaction(String seed, String destinationId,
       int value, int tick, int? inputType, String? payload) async {
-    CallAsyncJavaScriptResult? result = (inputType != null && payload != null)
-        ? await runFunction(QubicJSFunctions.createTransactionWithPayload, [
-            seed,
-            destinationId,
-            value.toString(),
-            tick.toString(),
-            inputType.toString(),
-            payload
-          ])
-        : await runFunction(QubicJSFunctions.createTransaction,
-            [seed, destinationId, value.toString(), tick.toString()]);
-
-    if (result == null) {
-      throw Exception(LocalizationManager
-          .instance.appLocalization.cmdErrorCreatingTransferTransactionGeneric);
-    }
-    if (result.error != null) {
-      throw Exception(LocalizationManager.instance.appLocalization
-          .cmdErrorCreatingTransferTransaction(result.error ?? ""));
+    try {
+      CallAsyncJavaScriptResult? result = (inputType != null)
+          ? await runFunction(QubicJSFunctions.createTransactionWithPayload, [
+              seed,
+              destinationId,
+              value.toString(),
+              tick.toString(),
+              inputType.toString(),
+              payload ?? " "
```

Comment:
@ahmed-tarek-salem is the method not accepting empty value for the payload? If that is the case, can we use "" ?

Reply:
For some reason "" fails, I'm investigating this.

Reply:
@sallymoc 

Fixed with 4b798be